libs/features/components/banners/src/lib/banners/elewa-group-image-and-text-banner/elewa-group-image-and-text-banner.component.html

```diff
@@ -0,0 +1,13 @@
+<div class="container">
+    <div class="content">
+        <p class="title1">{{title1}}</p>
+        <p class="description1">{{description1}}</p>
+    </div>
+    <div class="banner-img">
+        <img src="{{img1}}" class="banner-img" alt="">
+    </div>
+</div>
+
+
```

Comment:
please rename banners folder to components

Reply:
Done